### ISSUE TITLE
company proxy

### ISSUE THREAD

**Issue by gpagliuca** [binder-project/binder#47]
Hi there, great work. I wanted to show to some colleagues this possibility (this is a GREAT way to share code with non-python users) and my repo test works perfectly at home, but not at work. the notebook is opened correctly but the connection fails almost immediately. I strongly suspect this is related to the company proxy. Are you aware about that? Is there any workaround? Let me know if I can help maybe with some testing.
Again, great tool!

**Comment by rgbkrk** [comment]
If tmpnb.org or try.jupyter.org work for you, then it would likely get fixed with all the binder sites being switched to HTTPS on port 443.

**Comment by gpagliuca** [comment]
They work so it will be fine soon. Good news.

**Comment by andrewosh** [comment]
We recently switched the proxy server over to HTTPS, so the websockets should hopefully work fine behind most company proxies. I'll close this issue for now.

**Comment by gpagliuca** [comment]
Unfortunately it does not work for me. I tried with the same repository at home and at work and it works only at home. The problem is different now.

The page is not loaded with an "Unable to load the page" error page, both on Firefox and Chrome. I tried also on Linux (Chromium) and no difference, the app is not loaded when I am behind the proxy. It works outside.

I also tried changing the the default port (80) but without any result.
